public/utils/helpers.ts

```diff
@@ -45,3 +49,101 @@ export function diffJson(oldJson?: Record<string, any>, newJson?: Record<string,
   }
   return initial + addOrChanged + oldKeys.length;
 }
+
+// code related to filter blocked index/alias/datastream
+// an example to use:
+// import { aliasBlockedPredicate, filterBlockedItems } from "./helpers";
+// import { IndexOpBlocksType } from "./constants";
+// const result = filterBlockedItems<IAlias>(services, selectedItems, IndexOpBlocksType.Closed, aliasBlockedPredicate)
+
+interface BlockedIndices {
+  [indexName: string]: String[];
+}
+
+export async function getBlockedIndices(broserServices: BrowserServices): Promise<BlockedIndices> {
+  interface ClusterBlocksStateResponse {
```

Comment:
Why do we declare a type inside a function? It'd be better to move outside the function.

Reply:
Sure, I thought this interface is only used by this function. Will move it outside